invenio/base/templates/_formhelpers_base.html

```diff
@@ -23,7 +23,7 @@
   {% set label_size = fieldkwargs.get('label_size') %}
   {% set field_size = fieldkwargs.get('field_size') %}
   {% set with_label = fieldkwargs.pop('with_label', True) %}
-  {%- if field.name == "csrf_token" or field.type == 'HiddenField' -%}
+  {%- if field.type == "CSRFTokenField" or field.type == 'HiddenField' -%}
```

Comment:
We should find a different way to do this, since ``field.type == 'HiddenField'`` doesn't work for subclasses of ``HiddenField``.